platinum-bluetooth-device.html

```diff
@@ -92,36 +138,30 @@
         _device: {
```

Comment:
Could you expose the native `BluetoothDevice` so folks can get to any properties we add that haven't been added to the element yet?